includes/class-sensei-grading.php

```diff
@@ -652,22 +652,22 @@ public function admin_process_grading_submission() {
     } // end admin_process_grading_submission
 
 	public function get_redirect_url() {
-		// Parse POST data
-		$data = $_POST['data'];
-		$lesson_data = array();
-		parse_str($data, $lesson_data);
+		if ( ! isset( $_GET['course_id'] ) || ! isset( $_GET['lesson_id'] ) ) {
```

Comment:
Same here re: keeping the POST request and adding a nonce check.

Reply:
See https://github.com/Automattic/sensei/pull/2298#discussion_r231571585